docs/content/concepts/terminology.md

```diff
@@ -1,100 +1,142 @@
+
+## Exporting/Binding APIs
+
+One of the core values of the kcp project is to enable providing APIs that can be consumed by multiple workspaces.
+This is done via exporting/binding mechanism, i.e. via API Exports and API Bindings.
+
+The general workflow is:
+
+- Define your API resources similar to how you would define CustomResourceDefinitions (CRDs)
+- Create an API Export (in the same workspace) with a list of API resources that you want to export
+- User can create an API Binding in their workspaces referring to your API Export. This will result in API resources
+  that are defined in the referenced API Export to be installed in user's workspace
+- User can now create API resources of those types in their workspace. You can build and run controllers that are going
+  to reconcile those resources across different workspaces.
+
+The Workspace model defines one particular implementation of the lifecycle of a logical cluster and the APIs within it.
+Because APIs and the implementations that back an API evolve over time, it is important that the binding be
+introspectable and orchestrate-able - that a consumer can provide a rolling deployment of a new API or new
+implementation across hundreds or thousands of workspaces. The evolution of an API within a workspace and across
+workspaces is of key importance.
 
 ## Shard
 
 A failure domain within the larger control plane service that cuts across the primary functionality. Most distributed
-systems must separate functionality across shards to mitigate failures, and typically users interact with shards through
-some transparent serving infrastructure.  Since the primary problem of building distributed systems is reasoning about
-failure domains and dependencies across them, it is critical to allow operators to effectively match shards, understand
-dependencies, and bring them together.
+systems must separate functionality across shards to mitigate failures, and typically users interact with shards
+through some transparent serving infrastructure. Since the primary problem of building distributed systems is reasoning
+about failure domains and dependencies across them, it is critical to allow operators to effectively match shards,
+understand dependencies, and bring them together.
 
 A control plane should be shardable in a way that maximizes application SLO - gives users a tool that allows them to
 better define their applications not to fail.
 
-## API Binding
-
-The act of associating a set of APIs with a given logical cluster.  The Workspace model defines one particular
-implementation of the lifecycle of a logical cluster and the APIs within it.  Because APIs and the implementations that
-back an API evolve over time, it is important that the binding be introspectable and orchestrate-able - that a consumer
-can provide a rolling deployment of a new API or new implementation across hundreds or thousands of workspaces.
+## Index (e.g. Workspace Index)
```

Comment:
this thing is actually gone from kcp core.

Reply:
And it'll be gone from the document once I push some more fixes. :D